integration_tests/suite/test_api.py

```diff
@@ -233,6 +240,27 @@ def asterisk_reload_events_are_sent(request_uuid):
             asterisk_reload_events_are_sent, response['request_uuid'], timeout=5
         )
 
+    def test_exec_request_handlers_from_event(self):
+        asterisk_command = 'dialplan reload'
+        request_uuids = [str(uuid4())]
+        command_accumulator = self.bus.accumulator(
+            headers={'name': 'sysconfd_sentinel'}
+        )
+
+        self._send_bus_message(
+            'asterisk_reload_progress',
+            dict(origin_uuid='some-other-wazo'),
```

Comment:
Why did you use the `dict()` syntax here and `{}` on the following line? It feels like you're trying to play tricks with my brain